.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will do a clean install of node dependencies, build the source code and run lint across different versions of node
+name: Lint CI
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
```

Comment:
I think `lint` is better named for the task here. It will be visible in check and may confuse.

Reply:
I agree.